stytch/shared/jwt_helpers.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+import time
+from typing import Any, Dict, Optional
+
+import jwt as pyjwt
+import pydantic
+
+
+class GenericClaims(pydantic.BaseModel):
+    reserved_claims: Dict[str, Any]
+    untyped_claims: Dict[str, Any]
+
+
+def authenticate_jwt_local(
+    *,
+    jwks_client: pyjwt.PyJWKClient,
+    project_id: str,
+    jwt: str,
+    max_token_age_seconds: Optional[int] = None,
+    leeway: int = 0,
+) -> Optional[GenericClaims]:
```

Comment:
You know, it would be _really_ cool if we had a `stytch/shared/test/test_jwt_helpers.py` file :)
I think I'm empathetic if you don't want to do that right now, but could you at least go back and add a test in this PR summary to show that m2m auth + sessions auth are working as expected?